src/coreclr/src/vm/comutilnative.cpp

```diff
@@ -1273,9 +1273,8 @@ void QCALLTYPE GCInterface::_AddMemoryPressure(UINT64 bytesAllocated)
 {
     QCALL_CONTRACT;
 
-    // AddMemoryPressure could cause a GC, so we need a frame
     BEGIN_QCALL;
-    AddMemoryPressure(bytesAllocated);
+    NewAddMemoryPressure(bytesAllocated);
```

Comment:
Are there still existing uses of "AddMemoryPressure"?  If not, can we just rename NewAddMemoryPressure to AddMemoryPressure?

Reply:
Yep. Seems like that is the consensus. Done.